src/main/java/gift/model/category/Category.java

```diff
@@ -0,0 +1,76 @@
+package gift.model.category;
+
+import gift.model.gift.Gift;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "category")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(unique = true)
+    @NotNull
+    private String name;
+
+    @NotNull
+    private String color;
+
+    @Column(name = "imageurl")
+    @NotNull
+    private String imageUrl;
+
+    @NotNull
+    private String description;
+
+    @OneToMany(mappedBy = "category")
+    protected List<Gift> gifts = new ArrayList<>();
+
+    public Category() {
```

Comment:
여기서도 마찬가지로 protected로 사용하면 좋을 것 같습니다ㅎㅎ
```suggestion
    protected Category() {
```